LondonDataServices.IDecide.Core/Services/Orchestrations/Patients/PatientOrchestrationService.Exceptions.cs

```diff
@@ -70,6 +74,97 @@ private async ValueTask<Patient> TryCatch(
             }
         }
 
+        private async ValueTask TryCatch(
+            ReturningNothingFunction returningNothingFunction)
+        {
+            try
+            {
+                await returningNothingFunction();
+            }
+            catch (InvalidCaptchaException invalidCaptchaException)
+            {
+                throw await CreateAndLogValidationExceptionAsync(invalidCaptchaException);
+            }
+            catch (ValidPatientCodeExistsException validPatientCodeExistsException)
+            {
+                throw await CreateAndLogValidationExceptionAsync(
+                    validPatientCodeExistsException);
+            }
+            catch (InvalidPatientOrchestrationArgumentException invalidPatientOrchestrationArgumentException)
+            {
+                throw await CreateAndLogValidationExceptionAsync(invalidPatientOrchestrationArgumentException);
```

Comment:
Either wrap this line so you have a consistent reading flow or make all the others one-liners as they are all below the 120 limit.

So general rule is to maintain the code block for speed reading. If your wrapping is inconsistent then your middle line shifts where you have gaps or the code belly out.

```suggestion
                throw await CreateAndLogValidationExceptionAsync(
                    invalidPatientOrchestrationArgumentException);
```

Reply:
Fixed